cypress/helpers/data.helper.ts

```diff
@@ -18,15 +19,15 @@ export type VehicleFormElement = {
       readonly spaceshipForm: {
         readonly color: string;
         readonly canFire: boolean;
-        readonly numberOfPeopleOnBoard: number;
+        readonly crewMembersOnBoard: CrewMember[];
```

Comment:
While we're renaming things, I think just `crewMember` would be simpler, it is pretty irrelevant to the example whether they're on board or not

Reply:
Done in #68